Backend/app/dashboard/provider-dashboard/src/API/ProviderPlatform/DynamicOfferDriver/Driver.hs

```diff
@@ -549,3 +556,10 @@ updateSubscriptionDriverFeeAndInvoice merchantShortId apiTokenInfo driverId req
   transaction <- buildTransaction Common.UpdateSubscriptionDriverFeeAndInvoiceEndpoint apiTokenInfo driverId $ Just req
   T.withTransactionStoring transaction $
     Client.callDriverOfferBPP checkedMerchantId (.drivers.updateSubscriptionDriverFeeAndInvoice) driverId req
+
+sendSMSToDriverViaDashboard :: ShortId DM.Merchant -> ApiTokenInfo -> Id Common.Driver -> DDriver.SendSmsReq -> FlowHandler APISuccess
+sendSMSToDriverViaDashboard merchantShortId apiTokenInfo driverId req = withFlowHandlerAPI $ do
+  checkedMerchantId <- merchantAccessCheck merchantShortId apiTokenInfo.merchant.shortId
+  transaction <- buildTransaction Common.SendSmsToDriverViaDashboardEndPoint apiTokenInfo driverId (Just $ DDriver.VolunteerTransactionStorageReq apiTokenInfo.personId.getId driverId.getId (show req.messageKey))
```

Comment:
Why are we passing redundant data here ?

Reply:
@SrinivasNaikB  needed this data for analysis purpose. So I am storing the 
* volunteerId
* driverId 
* messageKey 

in the request of the transaction.